firehose.rng

```diff
@@ -121,7 +121,7 @@
              http://cwe.mitre.org/data/definitions/131.html
         -->
         <attribute name="cwe">
-          <data type="integer"/>
+          <data type="string"/>
         </attribute>
```

Comment:
As noted above, this probably should be something like:

    <zeroOrMore>
      <choice>
          <element name="cwe">
             <attribute name="cwe">
                <data type="integer"/>
             </attribute>
          </element>
          <element name="something-else"/>
      </choice>
    <zeroOrMore>

or somesuch.